pkgs/development/libraries/qt-6/default.nix

```diff
@@ -65,6 +67,11 @@ let
             url = "https://github.com/Homebrew/formula-patches/raw/c363f0edf9e90598d54bc3f4f1bacf95abbda282/qt/qt_internal_check_if_path_has_symlinks.patch";
             sha256 = "sha256-Gv2L8ymZSbJxcmUijKlT2NnkIB3bVH9D7YSsDX2noTU=";
           })
+          (substituteAll {
+            src = ./patches/qtbase-qmake-qt-prepare-tool.diff;
+            findQtTool = writeShellScript "find-qt-tool.sh" (
```

Comment:
todo: add `find-qt-tool` to `qt6.qtbase` passthru, so other tools can use it

todo: add `exec-qt-tool` for tools like `qtcreator`, see https://github.com/NixOS/nixpkgs/pull/189504#issuecomment-1427069057

exec-qt-tool

```sh
tool_file=$(find-qt-tool "$@")
if [ -n "$tool_file"]; then
  exec "$tool_file"
fi
echo "not found qt tool: $*" >&2
exit 127
```

Reply:
no, this does not work

qtcreator does not use `qtcreator-9.0.1/share/qtcreator/externaltools/qml.xml` to find qml